starters/gatsby-wordpress-starter/src/components/post.jsx

```diff
@@ -0,0 +1,74 @@
+import React from "react"
+
+import { Link } from "gatsby"
+import { GatsbyImage } from "gatsby-plugin-image"
+import parse from "html-react-parser"
+
+const Post = ({
+  post: { title, date, content, featuredImage },
+  previous,
+  next,
+}) => {
+  const imageData = {
+    gastbyImage:
+      featuredImage?.node?.localFile?.childImageSharp.gatsbyImageData,
+    altText: featuredImage?.node?.altText || title,
+  }
+
+  return (
+    <article className="prose lg:prose-xl mt-10 mx-auto">
+      <h1>{title}</h1>
+      <p className="text-sm text-gray-600">{new Date(date).toDateString()}</p>
+
+      <Link className="font-normal" to="/posts">
+        Posts &rarr;
+      </Link>
+
+      <div className="mt-12 max-w-lg mx-auto lg:grid-cols-3 lg:max-w-screen-lg">
+        {imageData.gastbyImage && (
+          <div className="relative w-full rounded-lg shadow-lg overflow-hidden mb-10">
+            <GatsbyImage
+              priority
+              image={imageData.gastbyImage}
```

Comment:
```suggestion
    gatsbyImage:
      featuredImage?.node?.localFile?.childImageSharp.gatsbyImageData,
    altText: featuredImage?.node?.altText || title,
  }

  return (
    <article className="prose lg:prose-xl mt-10 mx-auto">
      <h1>{title}</h1>
      <p className="text-sm text-gray-600">{new Date(date).toDateString()}</p>

      <Link className="font-normal" to="/posts">
        Posts &rarr;
      </Link>

      <div className="mt-12 max-w-lg mx-auto lg:grid-cols-3 lg:max-w-screen-lg">
        {imageData.gatsbyImage && (
          <div className="relative w-full rounded-lg shadow-lg overflow-hidden mb-10">
            <GatsbyImage
              priority
              image={imageData.gatsbyImage}
```

Reply:
My reading skill attacking again :p

Reply:
no worries, the typo was there before you started on this :)